scripts/ortoa-lib.sh

```diff
@@ -39,6 +47,50 @@ _EOF_
 }
 
 
+############################################
+# Running ORTOA
+############################################
+
+ortoa-client-run() {
```

Comment:
(discussion) would it be easier to use if we have a command `ortoa` with subcommands such as `client-run` and `simulate`. This can be left for later if we want to implement it

Reply:
If desired that should be a fairly easy wrapper to write. I'll leave it as a future addition.